lib/restforce/mash.rb

```diff
@@ -24,8 +24,11 @@ def klass(val)
           # of sobject records.
           Restforce::Collection
         elsif val.key? 'attributes'
-          if val['attributes']['type'] == 'Attachment'
+          case (val['attributes']['type'])
+          when /^Attachment$/
```

Comment:
A bit odd (and probably some overhead) to use a regular expression here, how about

```ruby
when "Attachment"
  ...
```